.gitignore

```diff
@@ -18,6 +18,7 @@
 /Library/PinnedKegs
 /Library/PinnedTaps
 /Library/Taps
+/Library/lock.d
```

Comment:
This looks a bit weird, given that we've managed to capitalize all directory names in there thus far. But I'm not feeling too strongly about this.